blackbelt/deployment.py

```diff
@@ -15,9 +15,9 @@ def deploy_production():
     post_message("Deploying to production", "#deploy-queue")
 
     slug_creaction_return_code = run_grunt_in_parallel((
-        ['grunt', 'create-slug'],
-        ['grunt', 'create-slug', '--app=apiary-staging-pre'],
-        ['grunt', 'create-slug', '--app=apiary-staging-qa'],
+        ['grunt', 'rollback'],
```

Comment:
I don't think this should be changed? Perhaps was done during testing? Will cause deploy to rollback instead.

Reply:
good catch, multicursor